apps/love-manager/values-summit.yaml

```diff
@@ -40,23 +40,115 @@ manager:
       initialDelaySeconds: 20
       periodSeconds: 10
   producers:
+  - name: general
     image:
       repository: ts-dockerhub.lsst.org/love-manager
-      tag: c0037.002
+      tag: c0037
```

Comment:
For the summit, we fix tags to a specific revision. Make sure this doesn't conflict or regress with what's on the summit.